src/layouts/hass-tabs-subpage.ts

```diff
@@ -26,6 +26,8 @@ export interface PageNavigation {
   component?: string;
   components?: string[];
   name?: string;
+  not_component?: string;
+  not_components?: string[];
```

Comment:
I'm open to name suggestions here, if there are any other alternatives that make more sense.

Reply:
we should just combine them 🙈 
`not_component?: string | string[];`

```js
Array.isArray(not_component) ? page.not_component.some((integration) => isComponentLoaded(hass, integration)) : isComponentLoaded(hass, page.not_component)
```
OR
```js
ensureArray(not_component).some((integration) => isComponentLoaded(hass, integration));
```

Reply:
Was just copying the current `component(s)` entries

Reply:
Yeah I know, they suck too 😄

Reply:
https://github.com/home-assistant/frontend/pull/19890